dev/docker-compose.yml

```diff
@@ -48,6 +48,7 @@ services:
       - "./common/settings.py:/etc/pulp/settings.py:z"
       - "${COMPOSE_CONTEXT}/..:/src:z"
       - "pulp_artifact:/var/lib/pulp/artifact"
+      - "ansible_tmp:/tmp/ansible"
```

Comment:
This is unnecessary. Normally you don't need persistent volume for temporary files.

Reply:
updated